src/types/public.ts

```diff
@@ -55,3 +55,14 @@ export interface TimeoutsOptions {
    */
   retries?: number;
 }
+
+export interface TaskEnablingOptions {
```

Comment:
nit: i think alignment with type naming conventions for the other types would be `TaskEnableOptions` (don't actually remember though)